### DISCUSSION TITLE
Scrum Prep Milestone 2 - Week 4

### DISCUSSION THREAD

**Discussion by nelson-escalona** [uprm-inso4101-2024-2025-s1/semester-project-regiupr#119] [Scrum Preps]
Oct 14 - Oct 18

**Comment by RayMuvz** [comment]
Finished implementing all features of the "Password Reset/Recovery"functionality of the program. Started working on Interface details (such as beautifying the screens and making it more user friendly)

**Comment by nelson-escalona** [comment]
Worked on doing documentation on some commands used in the EC2 Instance(how to ssh into it, how to copy files from a local machine to the instance, etc), worked on adding _on cascade delete_ to the database design, added a dummy user with the RegiUPR Gmail to the database, and I'm currently working on issue #84. The plan is to have it done by Sunday.

**Comment by J3SSY-ANDU** [comment]
Created 3 UI base features, but I haven't yet worked on them this week.

**Comment by cristianMarcial** [comment]
Curriculum Team. The text of each row of the scraped table was successfully extracted with the course section catalog, but all in a single list, which was the first step to work with the parser that my team is going to develop.

**Comment by keisanabria** [comment]
Did last friday's (11/oct) minute for the scrum meeting and will be doing this friday's (18/oct) scrum meeting's minute.

**Comment by PedroRodz** [comment]
Not much. Did a bit of work on tasks.

**Comment by carlospepin23** [comment]
Completed task #**97**. Next week, I'll be working with task #**96**, and create a Lecture Topic Task.

**Comment by nicolle-feliciano** [comment]
Couldn’t do much nor close tasks, I was preparing for a test I had yesterday. Will be available from now onwards to work on the tasks assigned by the team leader.

**Comment by Yamil-Serrano** [comment]
On this week, I encountered some health-related issues that temporarily prevented me from working on the project. In addition, I faced technical difficulties while trying to run the project in Visual Studio Code due to library installation failures, which caused further delays.

The last few weeks have also been particularly demanding as I had to allocate additional time to studying for my Circuit Analysis and Algorithm Design courses, which included preparing for exams and completing a project for the Algorithm Design class.

As a result, I was only able to complete one task, Role Definition and Privilege Mapping for the RBAC System #89, and opened another task today, Define Privileges for Roles: Admin and Student #121.

Moving forward, I hope to do my best, open more tasks, and complete a Lecture Topic task to contribute more effectively to the project.

**Comment by christianmedina-dev** [comment]
Login Team. Developed and implemented the Sign Up Form, also researched on sequence diagrams.

**Comment by Carloscaban** [comment]
Login team, already finished tasks for sprint, so I worked on other classes.
